politeiawww/wsdcrdata.go

```diff
@@ -123,8 +177,86 @@ func newWSDcrdata() (*wsDcrdata, error) {
 	log.Infof("Dcrdata pubsub server version: %v, client version %v",
 		serverSemVer, clientSemVer)
 
+	return c, nil
+}
+
+// reconnect creates a new websocket client, and subscribes to the
+// same subscriptions as the previous client.
+func (w *wsDcrdata) reconnect() error {
+	if w.isShutdown() {
+		log.Errorf("wsDcrdata is shutdown, not reconnecting")
```

Comment:
Don't log this here.  The error is being returned so its up to the calling function on whether or not to log it.